MoltenVK/MoltenVK/GPUObjects/MVKBuffer.mm

```diff
@@ -28,6 +28,16 @@
 #pragma mark -
 #pragma mark MVKBuffer
 
+void MVKBuffer::propogateDebugName() {
+	if (_deviceMemory &&
+		_deviceMemory->isDedicatedAllocation() &&
+		_deviceMemory->_debugName.length == 0) {
+
+		_deviceMemory->setDebugName(_debugName.UTF8String);
+	}
```

Comment:
`setDebugName()` is called with NULL pointer for dedicated allocation in `MVKBuffer::propogateDebugName()`.